### ISSUE TITLE
Diagnostics should not be OK if the realtime control loop is overrun

### ISSUE THREAD

**Issue by haudren** [Kawasaki-Robotics/khi_robot#46]
Hello,

I noticed the following behavior today during some testing:
- Connect to a running F60 controller attached to a RS007N
- Diagnostics are published correctly, showing "OK" as message and the diagnostics status is OK (0)
- Turn off the robot controller
- Message turns to "Realtime loop used too much time in the last 30 seconds.";
- Status stays OK (0)

I guess the problem is here:
https://github.com/Kawasaki-Robotics/khi_robot/blob/99cf142c46892a716ee6c9bbc0518e97fc803f12/khi_robot_control/src/main.cpp#L175

I would expect to see a 1 (WARNING) or 2 (ERROR) or even 3 (STALE)

By the way, I would recommend using Enums, as in 
```
diagnostics_msgs::DiagnosticStatus::WARN
```
Instead of directly using the values

**Comment by d-nakamichi** [comment]
Yes, currently status.level changes to 0 when overrun continues.
So I try to modify its level and overrun counting.

https://github.com/d-nakamichi/khi_robot/commits/modify_diagnostics_level

It would be helpful if you comment something about this.
